.eslintignore

```diff
@@ -1,2 +1,5 @@
 **/*_pb.*
 .eslintrc.js
+
+# Ignore WASM generated files:
+web/packages/teleport/src/ironrdp/pkg/**
```

Comment:
It needn't be in the gitignore because wasm-pack always generates this directory with a gitignore inside with

```
*
```

I did remove the ** from here and .prettierignore and everything still works.